### PR TITLE
[Web] add cloud strategies to community

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: Services/Interfaces/web_interface/controllers/welcome.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import flask
+
+import tentacles.Services.Interfaces.web_interface.login as login
+
+
+def register(blueprint):
```
Comment (Herklos) [inline Services/Interfaces/web_interface/controllers/welcome.py:21]: 👍

File: Services/Interfaces/web_interface/models/trading.py [hunk 2]
```diff
@@ -235,12 +235,22 @@ def get_pnl_history(exchange=None, quote=None, symbol=None, since=None, scale=No
     quote = None if symbol else quote
     history_by_exchange = _get_pnl_history(exchange, quote, symbol, since)
     invalid_pnls = 0
+    i = -300
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/trading.py:238]: ?

File: Services/Interfaces/web_interface/models/trading.py [hunk 3]
```diff
@@ -235,12 +235,22 @@ def get_pnl_history(exchange=None, quote=None, symbol=None, since=None, scale=No
...
+                if close_time < 1692835200:
+                    close_time += 3 * 31 * 27 * 3600
                 scaled_time = close_time - (close_time % scale_seconds)
                 pnl, pnl_p = historical_pnl.get_profits()
+                pnl *= i
+                pnl_p *= i
+                if pnl > 0:
+                    pnl *= 2
+                i *= -1
+                #tmp
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/trading.py:253]: ?

File: Services/Interfaces/web_interface/templates/profiles_selector.html [hunk 4]
```diff
@@ -34,43 +35,56 @@ <h3>
...
-                    <span class="badge badge-danger">without selected bot</span>
-                {% endif %}
+        <div>
+            <div>
+                <ul class="nav nav-tabs md-tabs justify-content-center" id="tabs" role="tablist">
+                    <li class="nav-item">
+                        <a class="nav-link primary-tab-selector {{ '' if use_cloud else 'active show' }}" id="your-profiles-tab" data-toggle="tab" href="#your-profiles" role="tab"
+                           aria-controls="your-profiles"
+                           aria-selected="{{ 'false' if use_cloud else 'true' }}">
+                            <h5>Base & custom</h5>
```
Comment (Herklos) [inline Services/Interfaces/web_interface/templates/profiles_selector.html:45]: What about "built-in" or "Default" or "standard" ?

Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/templates/profiles_selector.html:45]: i like "Default"

File: Services/Interfaces/web_interface/models/trading.py (hunk 3 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/trading.py:253]: woops

File: Services/Interfaces/web_interface/models/trading.py (hunk 2 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/trading.py:238]: woops

**Comment by GuillaumeDSM** [comment]
up !

**Review by Herklos — APPROVED** [review]

**Pull request merged**